config.yaml

```diff
@@ -39,6 +39,12 @@ build_iso:
      CentOS:
       '7': "./mock_configs/CentOS/7/build-iso-CentOS-7-ppc64le.cfg"
     mock_args:
+      # If the system has less RAM than required, the plugin will be disabled
+      "--enable-plugin=tmpfs
+      --plugin-option=tmpfs:keep_mounted=True
+      --plugin-option=tmpfs:max_fs_size=32g
+      --plugin-option=tmpfs:required_ram_mb=39800
```

Comment:
- I assume this change aims performance increase, pls document this in commit longer msg
- Certainly 32G for max FS size and 39800 MB for RAM is too much for ISO creation